test_istgt.sh

```diff
@@ -570,14 +570,28 @@ check_healthy_replica_count()
 {
```

Comment:
let's remove the wait time that is being handled in the caller of `check_healthy_replica_count`.